<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFVideoPlayerClass</class>
 <widget class="QMainWindow" name="FFVideoPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FFVideoPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VideoWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>611</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Playtime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    color: white;
	background-color: rgb(53,53,53);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MySlider" name="timerSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>641</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderVolume">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayVideo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCutImage">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSetVolume">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFullScreen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_OpenLocalFiles"/>
    <addaction name="action_OpenNetStream"/>
    <addaction name="action_OpenCamera"/>
   </widget>
   <widget class="QMenu" name="menushezhi">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_GetSourceCode"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menushezhi"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_OpenLocalFiles">
   <property name="text">
    <string>打开本地视频</string>
   </property>
  </action>
  <action name="action_OpenNetStream">
   <property name="text">
    <string>打开网络流</string>
   </property>
  </action>
  <action name="action_OpenCamera">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="action_GetSourceCode">
   <property name="text">
    <string>获取源代码</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>VideoWidget.h</header>
  </customwidget>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>MySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="FFVideoPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
